types/vimeo__player/index.d.ts

```diff
@@ -295,7 +308,11 @@ export class Player {
     on(event: string, callback: EventCallback): void;
     off<EventName extends keyof EventMap>(event: EventName, callback: EventCallback<EventMap[EventName]>): void;
     off(event: string, callback?: EventCallback): void;
-    loadVideo(id: number | string): VimeoPromise<number, TypeError | PasswordError | PrivacyError | Error>;
+    loadVideo(id: number): VimeoPromise<number, TypeError | PasswordError | PrivacyError | Error>;
+    loadVideo(url: string): VimeoPromise<string, TypeError | PasswordError | PrivacyError | Error>;
+    loadVideo(
+        options: Options,
+    ): VimeoPromise<{ [prop: string]: any }, TypeError | PasswordError | PrivacyError | Error>;
```

Comment:
Added overloads for the various versions of loadVideo. After some testing, it also seems that the resolved value is conditional depending on the type of argument passed.